nixos/modules/services/networking/6tunnel.nix

```
@@ -0,0 +1,111 @@
+{ lib, config, pkgs, ... }:
+
+with lib;
+
+let
+  cfg = config.services._6tunnel;
+
+in {
+  options.services._6tunnel = {
+
+    tunnels = mkOption {
+      default = [];
+      description = "Which tunnels to create.";
+      type = with types; listOf (submodule {
+        options = {
+          port = mkOption {
+            description = "Local port to open.";
+            type = types.port;
+          };
+          host = mkOption {
```

I prefer
```suggestion
          remoteHost = mkOption {
```
and that's also what the man page calls it